{
  "id": "GHSA-f9xv-q969-pqx4",
  "summary": "Uncaught Exception in yaml",
  "details": "Uncaught Exception in GitHub repository eemeli/yaml starting at version 2.0.0-5 and prior to 2.2.2.",
  "aliases": [
    "CVE-2023-2251"
  ],
  "modified": "2023-05-03T18:36:52Z",
  "published": "2023-04-24T15:30:34Z",
  "database_specific": {
    "nvd_published_at": null,
    "github_reviewed_at": "2023-04-24T18:05:30Z",
    "severity": "HIGH",
    "github_reviewed": true,
    "cwe_ids": [
      "CWE-248"
    ]
  },
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://nvd.nist.gov/vuln/detail/CVE-2023-2251"
    },
    {
      "type": "WEB",
      "url": "https://github.com/eemeli/yaml/commit/984f5781ffd807e58cad3b5c8da1f940dab75fba"
    },
    {
      "type": "PACKAGE",
      "url": "https://github.com/eemeli/yaml"
    },
    {
      "type": "WEB",
      "url": "https://huntr.dev/bounties/4b494e99-5a3e-40d9-8678-277f3060e96c"
    }
  ],
  "affected": [
    {
      "package": {
        "name": "yaml",
        "ecosystem": "npm",
        "purl": "pkg:npm/yaml"
      },
      "ranges": [
        {
          "type": "SEMVER",
          "events": [
            {
              "introduced": "2.0.0-5"
            },
            {
              "fixed": "2.2.2"
            }
          ]
        }
      ],
      "database_specific": {
        "source": "https://github.com/github/advisory-database/blob/main/advisories/github-reviewed/2023/04/GHSA-f9xv-q969-pqx4/GHSA-f9xv-q969-pqx4.json"
      }
    }
  ],
  "schema_version": "1.6.0",
  "severity": [
    {
      "type": "CVSS_V3",
      "score": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H"
    }
  ]
}